sdk/keyvault/azure-resourcemanager-keyvault/src/test/java/com/azure/resourcemanager/keyvault/VaultTests.java

```diff
@@ -25,93 +19,91 @@ public class VaultTests extends KeyVaultManagementTest {
     @Test
     public void canCRUDVault() throws Exception {
         // Create user service principal
-        String sp = generateRandomResourceName("sp", 20);
-        String us = generateRandomResourceName("us", 20);
-        ServicePrincipal servicePrincipal
-            = authorizationManager.servicePrincipals().define(sp).withNewApplication().create();
+        //        String sp = generateRandomResourceName("sp", 20);
+        //        String us = generateRandomResourceName("us", 20);
+        // issue: https://github.com/Azure/azure-sdk-for-java/issues/47117
+        //        ServicePrincipal servicePrincipal
+        //            = authorizationManager.servicePrincipals().define(sp).withNewApplication().create();
 
-        ActiveDirectoryUser user
-            = authorizationManager.users().define(us).withEmailAlias(us).withPassword(password()).create();
+        // Status code 403, "{"error":{"code":"Authorization_RequestDenied","message":"Insufficient privileges to complete the operation."
+        //        ActiveDirectoryUser user
+        //            = authorizationManager.users().define(us).withEmailAlias(us).withPassword(password()).create();
```

Comment:
Change to tests are mostly about this.
1. `Application` needs an additional `serviceTreeID` parameter
2. We don't have permission to create AAD user anymore.